packages/amplify/amplify_flutter/README.md

```diff
@@ -4,7 +4,9 @@ The top level module for Amplify Flutter.
 
 ## Developer Preview
 
-This package is part of the the developer preview for the Amplify Auth category on mobile, web and desktop. It is not intended for production usage. 
+This package version is part of the the developer preview for the Amplify Auth category on mobile, web and desktop. It is not intended for production usage. 
```

Comment:
Should we just say 
```suggestion
This package version is part of our developer preview for mobile, web and desktop platforms. It is not intended for production usage. 
```

Otherwise, you'll need to update this every time we release a category and I'm not sure it makes much sense to say things like `amplify_api` are part of the Amplify Auth dev preview.

Reply:
@dnys1 Makes sense to me.